distribution/lib/Standard/Base/0.0.0-dev/src/Errors/Common.enso

```diff
@@ -476,3 +477,19 @@ type Additional_Warnings
     to_display_text : Text
     to_display_text self =
         "There were "+self.count.to_text+" additional issues."
+
+type Loss_Of_Numeric_Precision
+    ## PRIVATE
+       Indicates that a numeric conversion of a value has lost precision.
+    Warning (original_value : Any) (new_value : Any)
```

Comment:
Given the Any isn't doing anything shall we remove and avoid import?

Reply:
Done.